### PR TITLE
Rephrase some strings to a less technical language

### PR DESCRIPTION
**Pull request by delight-aug**
Rephrase the paragraph about GUI apps and Apx to not use technical terms and generally to read better. I'm not sure if I should leave `"Open with"` with quotes unlike `Application menu`, use quotes for both, or for neither.

I'm also not sure whether to change `display servers` to `displays` in the list of things Apx share from host to containers. I really don't like the `display server` term, it's not accurate even as a technical one.

### CODE REVIEW

File: _posts/2022-12-29-vanilla-os-22-10-kinetic.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ conflicting packages.
 
 It achieves this by installing software inside one or more containers fully
 managed by Apx having restricted access to your system’s resources while still
-being able to use the same drivers, display server, etc.
+being able to use the same drivers, displays, etc.
```
Comment (kbdharun) [inline _posts/2022-12-29-vanilla-os-22-10-kinetic.md:47]:
```suggestion
being able to use the same drivers, hardware, graphical apps, etc.
```
This is the one used for distrobox. I seem to have missed this when writing the docs.

File: _posts/2022-12-29-vanilla-os-22-10-kinetic.md [hunk 2]
```diff
@@ -62,11 +62,9 @@ system.
 
 ![Apx Control Center](/assets/uploads/vanilla-os-apx-control-center.png)
 
-For GUI packages created inside apx containers, .desktop files are created 
-automatically and added to the Applications menu. These applications are also 
-displayed beside other applications in the “Open with” menu in nautilus. 
-GUI Packages installed inside containers get shown in the Sub System section 
-of Vanilla control center.
+Graphical applications installed inside apx containers are automatically added 
+to the Applications menu as well as the “Open with” menu in the file manager.
```
Comment (kbdharun) [inline _posts/2022-12-29-vanilla-os-22-10-kinetic.md:66]:
```suggestion
to the Applications menu as well as the “Open with” menu in the file manager (`nautilus`). 
```

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution. I have some suggestions for this page.

After this PRs merge I will update the docs too.

File: _posts/2022-12-29-vanilla-os-22-10-kinetic.md (hunk 2 above)
Comment (delight-aug) [reply _posts/2022-12-29-vanilla-os-22-10-kinetic.md:66]: I guess you really like nautilus 😛 There's really no reason to clutter the sentence with this. The name 'nautilus' is only ever used as a codename, it's never exposed to the user.

File: _posts/2022-12-29-vanilla-os-22-10-kinetic.md (hunk 1 above)
Comment (delight-aug) [reply _posts/2022-12-29-vanilla-os-22-10-kinetic.md:47]: You can't run host GUI applications from inside the container, can you?

Comment (kbdharun) [reply _posts/2022-12-29-vanilla-os-22-10-kinetic.md:47]: > You can't run host GUI applications from inside the container, can you?

Yeah, you can't in distrobox they are meaning Wayland/X11. But I guess we can leave the hardware alone.

File: _posts/2022-12-29-vanilla-os-22-10-kinetic.md (hunk 2 above)
Comment (kbdharun) [reply _posts/2022-12-29-vanilla-os-22-10-kinetic.md:66]: > I guess you really like nautilus 😛 There's really no reason to clutter the sentence with this. The name 'nautilus' is only ever used as a codename, it's never exposed to the user. 

Agreed, I will mark this as resolved 🙂

File: _posts/2022-12-29-vanilla-os-22-10-kinetic.md (hunk 1 above)
Comment (delight-aug) [reply _posts/2022-12-29-vanilla-os-22-10-kinetic.md:47]: You mean both drivers and hardware? That sounds good to me.

Comment (kbdharun) [reply _posts/2022-12-29-vanilla-os-22-10-kinetic.md:47]: > You mean both drivers and hardware? That sounds good to me. 

Sure I guess.

**Pull request merged**